File: Entidades/Bodega.vb
Public Class Bodega

    Private _varIdProducto As Integer
    Public Property Id_Producto() As Integer
        Get
            Return _varIdProducto
        End Get
        Set(ByVal value As Integer)
            _varIdProducto = value
        End Set
    End Property

    Private _varExistBodega As Double
    Public Property ExistBodega() As Double
        Get
            Return _varExistBodega
        End Get
        Set(ByVal value As Double)
            _varExistBodega = value
        End Set
    End Property


    Private _varMaximoCaja As Double
    Public Property MaximoCaja() As Double
        Get
            Return _varMaximoCaja
        End Get
        Set(ByVal value As Double)
            _varMaximoCaja = value
        End Set
    End Property

End Class

File: Entities/Bodega.cs
namespace Entidades
{
    using System;

    public class Bodega
    {
        private int _varIdProducto;
        private double _varExistBodega;
        private double _varMaximoCaja;

        public int Id_Producto
        {
            get => 
                this._varIdProducto;
            set => 
                this._varIdProducto = value;
        }

        public double ExistBodega
        {
            get => 
                this._varExistBodega;
            set => 
                this._varExistBodega = value;
        }

        public double MaximoCaja
        {
            get => 
                this._varMaximoCaja;
            set => 
                this._varMaximoCaja = value;
        }
    }
}

